share/completions/sad.fish

```diff
@@ -0,0 +1,9 @@
+complete sad    -s 0 -l read0                    -d 'Use \0 as stdin delimiter'
+complete sad    -s e -l exact                    -d 'String literal mode'
+complete sad -x -s f -l flags                    -d "Regex flags: use `--help` instead of `-h` to see details"
+complete sad -x      -l fzf     -a "never"       -d "Additional Fzf options, disable = never"
```

Comment:
What does this disable, exactly?

Reply:
including `--fzf=never` disables fzf and previews the changes with `$GIT_PAGER`.

Reply:
And passing other values gives options to fzf?

Reply:
Yes, e.g. try this in `fish-shell/share/completions/`: `ls sad.fish | sad --fzf="--header 'lol?'" "complete" "nope"` overwrites the fzf header.